src/js/utils/dateUtils.js

```diff
@@ -17,4 +27,149 @@ const dateWithoutTimeZone = ({
   return parsedDate.format(outputDateFormat);
 };
 
+export const parseDateAndStripTimezone = (date, providedFormat) => {
+  const dateWithoutOffset = date.replace(/([+-]\d{2}:\d{2}|Z)$/, '');
```

Comment:
what do we mean by "offset" here?

Reply:
this part of dates:
2025-10-02T08:15:30 **+01:00**
2025-10-02T12:45:00 **+02:00**
2025-10-02T19:05:10 **-04:00**